main.pm

```diff
@@ -420,7 +424,7 @@ sub load_consoletests() {
         loadtest "console/consoletest_setup.pm";
         loadtest "console/textinfo.pm";
         loadtest "console/hostname.pm";
-        if (get_var("FILESYSTEM") == "btrfs") {
```

Comment:
yeah, my mistake review for https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/462 ...